### DISCUSSION TITLE
WalletConnectSwiftV2 ExampleApp cannot pair

### DISCUSSION THREAD

**Discussion by livid** [WalletConnect/walletconnect-monorepo#1596] [Developer Support]
I always get this error both on the Simulator and on a real device.

`webSocketNotConnected`

<img width="904" alt="Screenshot 2022-11-04 at 2 10 58 AM" src="https://user-images.githubusercontent.com/11427/199936266-09ebecd1-8383-4a57-8449-1026c1d6d638.png">

In `Configuration.xcconfig` there is a `RELAY_HOST` config:

```
RELAY_HOST = relay.walletconnect.com
```

I added my `PROJECT_ID` in `Configuration.xcconfig`. What else should I do to get the example app up and running?

**Comment by llbartekll** [comment]
hey, it is usually something with project id. Maybe you are providing it as a string surrounded by `" "`?
if so, remove `" "`

**Reply by livid** [reply]
It seems to relate to `Allowed Domains`, I need to leave them empty for the example app.

**Reply by chadyj** [reply]
@livid What value did you have in there?
What was your expectation?

**Reply by livid** [reply]
I had two items in `Allowed Domains` like this:

<img width="901" alt="Screenshot 2022-11-04 at 4 23 47 PM" src="https://user-images.githubusercontent.com/11427/200088698-6b2b1a3a-6e4b-468c-a35d-834b024d29ad.png">

After I cleared them, I can use the PROJECT_ID to run the example app.
